tldw_Server_API/app/api/v1/endpoints/audio.py

```
@@ -46,6 +47,16 @@
     bytes_to_seconds,
 )
+# Quota helpers for status/limits and TTL heartbeat
+try:
+    from tldw_Server_API.app.core.Usage.audio_quota import (
+        heartbeat_stream as heartbeat_stream,
+        active_streams_count as active_streams_count,
+        get_daily_minutes_used as get_daily_minutes_used,
+        get_user_tier as get_user_tier,
+    )
+except Exception:
+    pass
```

Ruff (0.14.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)